contracts/sol6/KyberNetwork.sol

```diff
@@ -865,10 +867,22 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         IERC20 src,
         IERC20 dest,
         uint256 srcAmount,
-        uint256 networkFeeBps,
-        uint256 networkFeeValue
+        TradeData memory tradeData
     ) internal view returns (uint256[] memory feesAccountedDestBps) {
         uint256 numReserves = reservesData.ids.length;
+        uint256 srcAmountAfterFee;
+        uint256 destAmountFeeBps;
+
+        if (src == ETH_TOKEN_ADDRESS) {
+            //+ srcAmountAfterFee < MAX_QTY
+            srcAmountAfterFee = srcAmount - 
+                (tradeData.tradeWei * tradeData.networkFeeBps / BPS);
+                require(srcAmountAfterFee < srcAmount, "high fees");
```

Comment:
should we change the msg to `underflow subtraction`. And add a comment that it is a safe check ( can not reach from our code)

Reply:
removing message